### PR TITLE
[FIX] fix issue 9194 auth_signup password reset email is always sent …

### PR DESCRIPTION
**Pull request by cvinh**
…in english

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Have you made the upstream PR on Odoo?

**Comment by cvinh** [comment]
yes PR has been made on odoo/odoo

**Comment by pedrobaeza** [comment]
Please put here the link

**Comment by cvinh** [comment]
https://github.com/odoo/odoo/pull/9195

**Comment by pedrobaeza** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
Thanks for updating the PR to follow up on my remarks! Next time, please add a comment saying that you did so, otherwise I am not notified of the updates in the code and I may not come back to this PR for a long time.

File: addons/auth_signup/res_users.py [hunk 1]
```diff
@@ -292,6 +291,7 @@ def action_reset_password(self, cr, uid, ids, context=None):
         for user in self.browse(cr, uid, ids, context):
             if not user.email:
                 raise osv.except_osv(_("Cannot send email: user has no email address."), user.name)
+            context={'lang': user.lang}
```
Comment (StefanRijnhart) [inline addons/auth_signup/res_users.py]: Oh, please preserve the rest of the context:

```
    context['lang'] = user.lang
```

**Comment by cvinh** [comment]
thanks Stefan, I updated the PR

**Comment by StefanRijnhart** [comment]
Thanks! :+1:

**Pull request merged**